modules/deployments-ab-testing-lb.adoc

```diff
@@ -113,23 +113,36 @@ spec:
 ----
 
 [id="deployments-ab-testing-lb-web_{context}"]
-== Managing weights using the web console
+== Managing weights of an existing route using the web console
 
 .Procedure
 
-. Navigate to the Route details page (Applications/Routes).
+. Navigate to the *Networking* -> *Routes* page.
 
-. Select *Edit* from the Actions menu.
+. Click the Actions menu {kebab} next to the Route you want to edit and select *Edit Route*.
 
-. Check *Split traffic across multiple services*.
+. Edit the YAML file. Update the `weight` to be an integer between `0` and `256` that specifies the  target's relative weight against other target reference objects. The value `0` suppresses requests to this back-end. The default is `100`. Run `oc explain routes.spec.alternateBackends` for more information about the options.
```

Comment:
s/back-end/back end because it is a noun here

Reply:
s/target's relative weight/relative weight of the target/ (ISG suggests avoiding contractions)